dart/README.md

```diff
@@ -54,8 +57,10 @@ Future<void> main() async {
       stackTrace: stackTrace,
     );
   }
+  // ...
```

Comment:
why do we need this?

Reply:
to illustrate that the app runner will probably contain more code.
What's the better : `// the rest of your code` or nothing ?

Reply:
✅